tests/compliance/date/test_date_compliance_1_5.py

```diff
@@ -24,7 +24,7 @@
 import pytest
 
 # NDArrayBacked2DTests suite added in https://github.com/pandas-dev/pandas/pull/44974
-pytest.importorskip("pandas", minversion="1.5.0dev")
+pytest.importorskip("pandas")
```

Comment:
`pandas` is a strict dependency. I don't think we need this line. We were just using it for the min version before.

```suggestion
```

Reply:
Since we rely on `pandas` version `>=1.5.0` this whole file feels unnecessary. I am gonna move this test to the generic compliance file as opposed to this version specific file. <fingers crossed>

Reply:
Done.